src/components/Reactions/hooks/useProcessReactions.tsx

```diff
@@ -86,8 +86,21 @@ export const useProcessReactions = <
     [reactionCounts, supportedReactionsArePresent],
   );
 
+  const aggregatedNamesByType = useMemo(
+    () =>
+      latestReactions.reduce<Record<string, Array<string>>>((pv, { type, user }) => {
+        pv[type] ??= [];
```

Comment:
@arnautov-anton just nitpicking but what `pv` stands for?

Reply:
`pv` as in "previous value". It's something I've started with and it stuck with me. I like it more than `acc` which in this case is more suitable but never sat right with me. Might start using `av` as in "accumulated value". :smile:

Reply:
I was also tripped by the variable naming here.